PinholeCamera symbols not exported

## Description
I want to use PinholeCamera from gtsam, but I have linking error:

gtsam.lib(gtsam.dll) : error LNK2005: "public: __cdecl gtsam::PinholeCamera<class gtsam::Cal3_S2>::PinholeCamera<class gtsam::Cal3_S2>(class gtsam::Pose3 const &,class gtsam::Cal3_S2 const &)" (??0?$PinholeCamera@VCal3_S2@gtsam@@@gtsam@@QEAA@AEBVPose3@1@AEBVCal3_S2@1@@Z) already defined in my_lib.lib(File.obj) [path_to_unit_tests.vcxproj]

## Steps to reproduce

1. Build shared gtsam
2. Use PinholeCamera in your source
3. Build your code in shared configuration

<!-- If you have a code sample, unit test, error messages, stack traces, etc., please provide it here as well -->

## Expected behavior

Linking properly

## Environment

C++17, VS2017

## Additional information

I already fixed this issue. There are 2 ways to do it:
a) use SimpleCamera.h, but it is deprecated:
```
/**
 * @deprecated: SimpleCamera for backwards compatability with GTSAM 3.x
 * Use PinholeCameraCal3_S2 instead
 */
class GTSAM_EXPORT SimpleCamera : public PinholeCameraCal3_S2 {
```
b) option wich in my opinion should be added to gtsam - change PinholeCamera.h:
from:
`class PinholeCamera: public PinholeBaseK<Calibration> {`
to
`class GTSAM_EXPORT PinholeCamera: public PinholeBaseK<Calibration> {`

--- a/gtsam/geometry/PinholeCamera.h
+++ b/gtsam/geometry/PinholeCamera.h
@@ -30,7 +30,7 @@ namespace gtsam {
  * \nosubgrouping
  */
 template<typename Calibration>
-class PinholeCamera: public PinholeBaseK<Calibration> {
+class GTSAM_EXPORT PinholeCamera: public PinholeBaseK<Calibration> {
 
 public:
 

--- a/gtsam/geometry/PinholePose.h
+++ b/gtsam/geometry/PinholePose.h
@@ -31,7 +31,7 @@ namespace gtsam {
  * \nosubgrouping
  */
 template<typename CALIBRATION>
-class PinholeBaseK: public PinholeBase {
+class GTSAM_EXPORT PinholeBaseK: public PinholeBase {
 
 private:
 

